decentraland/unity-renderer
# chore: Upgrade to Unity 2021.3.18f1

## What does this PR change?

This PR upgrade the version of Unity to the latest LTS minor release, which has some important improvement and fixes.

...

## How to test the changes?



1. Go to: https://play.decentraland.zone/?explorer-branch={branch_name}&{desired_url_params}
2. Do a general check, nothing in the code has been changed.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
